package.json

```diff
@@ -22,6 +22,7 @@
     "ethereum-ens": "^0.7.4",
     "ethereumjs-abi": "^0.6.5",
     "ethereumjs-testrpc-sc": "^6.1.2",
+    "glob": "^7.1.2",
```

Comment:
did this just get missed in a previous PR?

Reply:
no, it worked fine, but provided an nnoying eslint error without adding it. is there a better eslint rule solution? tech dont need it in deps